packages/lib/gpcircuits/test/common.ts

```diff
@@ -0,0 +1,59 @@
+import { PODEntries } from "@pcd/pod";
+import { Identity } from "@semaphore-protocol/identity";
+import { expect } from "chai";
+import { Circomkit } from "circomkit";
+import { readFileSync } from "fs";
+import path from "path";
+
+const configFilePath = path.join(__dirname, "../circomkit.json");
+const config = JSON.parse(readFileSync(configFilePath, "utf-8"));
```

Comment:
any reason not to have this configuration be an object exported from a typescript file?

Reply:
I didn't realize that it was necessary for `circomkit` when I made the suggestion, so feel free to disregard.